bazel/toolchains/regenerate.sh

```diff
@@ -4,9 +4,10 @@ set -e
 
 export RBE_AUTOCONF_ROOT=$(bazel info workspace)
 
-rm -rf "${RBE_AUTOCONF_ROOT}/bazel/toolchains/configs/*"
+rm -rf "${RBE_AUTOCONF_ROOT}"/bazel/toolchains/configs/*
 cp -vf "${RBE_AUTOCONF_ROOT}/bazel/toolchains/empty.bzl" "${RBE_AUTOCONF_ROOT}/bazel/toolchains/configs/versions.bzl"
 
 # Bazel query is the right command so bazel won't fail itself.
-bazel query "@rbe_ubuntu_clang_gen//..."
-bazel query "@rbe_ubuntu_gcc_gen//..."
+bazel query ${BAZEL_QUERY_OPTIONS} "@rbe_ubuntu_clang_gen//..."
+bazel query ${BAZEL_QUERY_OPTIONS} "@rbe_ubuntu_clang_libcxx_gen//..."
+bazel query ${BAZEL_QUERY_OPTIONS} "@rbe_ubuntu_gcc_gen//..."
```

Comment:
why is this checked in? I thought that referencing that repo was enough?

Reply:
(the files this script generates I mean)

Reply:
I don't want to generate config everytime CI runs, so just generate them and check in the config.

Reply:
https://github.com/bazelbuild/bazel-toolchains/blob/0.28.1/rules/rbe_repo.bzl#L44

Reply:
Ah thanks